beancount-parser/src/lib.rs

```diff
@@ -423,6 +485,9 @@ fn transaction_directive<'i>(
                         }
                     }
                 }
+                for t in state.get_tags() {
```

Comment:
Even though the for-loop body is only one line, I think `for tag in state.get_tags()` might read better?